invenio_i18n/babel.py

```diff
@@ -6,30 +6,31 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Flask-BabelEx domain for merging translations from many directories."""
-
-from __future__ import absolute_import, print_function
+"""Flask-Babel domain for merging translations from many directories."""
 
 import os
 from contextlib import contextmanager
 
-from babel.support import NullTranslations, Translations
-from flask import _request_ctx_stack, current_app
-from flask_babelex import Domain, get_locale
+from babel import Locale
+from flask_babel import Domain, _get_current_context
 from pkg_resources import iter_entry_points, resource_filename, resource_isdir
 
 
+# TODO:
+# think of replacing it with force_locale from flask_babel
```

Comment:
Good idea, please do.
On top of that, I cannot find any usage of `set_locale` in Invenio.

I also think that the previous code:
```
ctx = _request_ctx_stack.top
    if ctx is None:
        raise RuntimeError("Working outside of request context.")
```
was wrong, it was forcing to have a request context...

Reply:
I also didn't found any usage of this function. 

What do you mean by `was wrong`? I thought that this function makes only sense if there is a request context. It is also stated in the documentation of the function that the language is set in the request context.

Should we remove it and add to the release notes that if the function is used it should use flask-babel's force_locale instead?

Reply:
i removed it with [commit](https://github.com/inveniosoftware/invenio-i18n/pull/108/commits/82ca3d172006dbb16d4c196508ae505ecc7cceb5) but i let the tests as they are by replacing `set_locale` with `force_locale`